.grit/patterns/thousands_separator.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Add thousands separator
+---
+
+Add thousands separator (`1_000_000`) to numbers (ints and floats, positive or negative).
+
+```grit
+engine marzano(0.1)
+language python
+
+
+`$number` where {
+    $number <: or { integer(), float() },
```

Comment:
This is another example of the anti-pattern (maybe we should put this in the docs). If a variable is immediately restricted after assignment, start with the restriction.

```suggestion
or { integer(), float() } as number where {
```

Reply:
Fixed.